ADR/user-specified-tokens-build.md

```diff
@@ -0,0 +1,28 @@
+# Enable users to configure tokens/credentials for a Build/Test Pipeline
+
+Date: 2023-03-03
+
+## Status
+
+Provisional
+
+
+## Context
+
+As a user, I would like to configure sensitive information/tokens/credentials for the build/test pipeline to consume as part of a partner task.
```

Comment:
Is this explicitly for the use within partner tasks, or is it generically to be used by tests in the workspace (i.e. if credentials are needed for user-provided integration tests)?

Reply:
It would be both.

Reply:
That is awesome -- but I think that also means that we have to loosen the language from "partner task" in the ADR as that seems to be too narrowly focused. Can it just be generalized to "a part of a task"?

If we want to call out the fact that this enables the configuring and running of partner tasks, then can we relegate that wording/comment to the consequences?

Reply:
Does this help https://github.com/redhat-appstudio/book/pull/64/commits/1191aa8bca2e893fe1765be91f4c257bd7c3f673 to begin with ?

Reply:
It is helpful in that we mention integration tests somewhere now. Can we just strike `partner` from this line or add `or user`?